src/proposals/mips/mip-x03/MTokens.json

```diff
@@ -0,0 +1,44 @@
+{
+    "8453": [
+        {
+            "addressesString": "MOONWELL_rsETH",
```

Comment:
Please rename to `MOONWELL_wrsETH` to match the asset we are adding (wrapped rsETH)